src/Umbraco.Cms.Integrations.DAM.Aprimo/Services/AprimoService.cs

```diff
@@ -57,6 +62,10 @@ public async Task<AprimoResponse<Record>> GetRecordById(Guid id)
 
             client.DefaultRequestHeaders.Authorization = new AuthenticationHeaderValue("Bearer", accessToken);
 
+            client.DefaultRequestHeaders.Add("select-record", "title,thumbnail,fields,masterfilelatestversion");
```

Comment:
Consider moving these headers to the request's headers collection rather than the client's default request headers.  I'm not sure, but could be how you are doing it these headers would be used in another request unless overridden.  There's a few examples of this in this class.